README.markdown

```diff
@@ -22,4 +23,6 @@ Run a webserver in the output folder:
     $ cd _site/
```

Comment:
Oh so this might not have used to work, but just run `jekyll serve` to run a local testing websever. Please remove the notes about python.

https://jekyllrb.com/docs/usage/